<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonSelect">
     <property name="text">
      <string>Select Part</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSelected">
     <property name="text">
      <string>&lt;nothing selected&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonRed">
     <property name="text">
      <string>red</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioButtonGreen">
     <property name="text">
      <string>green</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
